tgg/research/gnomad_tgps.py

```diff
@@ -0,0 +1,354 @@
+    :return: None; function writes output to the output path.
+    """
+    with hfs.open(f"{file_dir}/{filename}", "w") as f:
+        print(var_stats, file=f)
+
+
+def compute_stats(
+    pangenome_ids_path: str,
+    output_dir: str,
+    overwrite: bool,
+) -> None:
+    """
+    Count the number of common (AF > 1%) variants per gnomAD sample.
+
+    Function produces counts only for gnomAD samples that are not part of the pangenome.
+
+    :param pangenome_ids_path: Path to text file (stored in Google bucket)
+    containing pangenome sample IDs.
+    :param output_dir: Path to a Google bucket for output file.
+    :param overwrite: A Boolean variable to specify whether or not to overwrite an
+    existing file.
+    :return: None; function writes a text file to output bucket.
+    """
+    # Get public gnomAD release sites HT.
+    gnomad_freq_ht = release_sites().ht().select_globals().select("freq")
+
+    # Get gnomAD v3 metadata and select two entry fields:
+    # release: indicates if the samples are released;
+    # population_inference: genetic ancestry information.
+    meta_ht = meta.ht().select_globals().select("release", "population_inference")
+    meta_ht = meta_ht.filter(meta_ht.release == True)
+
+    # Read gnomAD v3 variant dataset, select genotype entry field, and split
+    # multi-allelics.
+    mt = get_gnomad_v3_vds().variant_data
+    mt = mt.select_rows().select_entries("LA", "LGT")
+    mt = hl.experimental.sparse_split_multi(mt)
+
+    # Use the filtered metadata to filter gnomAD v3 data.
+    mt = mt.filter_cols(hl.is_defined(meta_ht[mt.col_key]))
+    num_of_v3_samples = mt.count_cols()
+    logger.info("The number of samples in gnomAD v3 is: %i", num_of_v3_samples)
+    if num_of_v3_samples != NUM_OF_GNOMAD_SAMPLES:
+        raise DataException(
+            f"Found {num_of_v3_samples} samples in VDS but was expecting"
+            f" {NUM_OF_GNOMAD_SAMPLES} samples; please double check!"
+        )
+
+    # Get pangenome sample IDs.
+    pangenome_ids = get_pangenome_ids(pangenome_ids_path)
+    logger.info(
+        "The total number of provided pangenome IDs is %i",
+        hl.eval(hl.len(pangenome_ids)),
+    )
+
+    # Filter mt to contain gnomAD samples not in pangenome.
+    mt = mt.filter_cols(~pangenome_ids.contains(mt.s))
+
+    # Annotate mt with allele frequencies and filter out rare SNVs (AF lower than 1%).
+    # Also filter out sites where no gnomAD samples had a variant call.
+    mt = mt.annotate_rows(freq=gnomad_freq_ht[mt.row_key].freq)
+    mt = mt.filter_rows(mt.freq.AF[0] > 0.01)
+    mt = mt.filter_rows(hl.agg.any(mt.GT.is_non_ref()))
+
+    # Read the HGDP+1KG subset matrix table, select the genotype entry field, split
+    # multi-allelics, and select SNVs.
+    pg_mt = hl.read_matrix_table(HGDP_SUBSET_MT)
+    pg_mt = pg_mt.select_rows().select_entries("LA", "LGT")
+    pg_mt = hl.experimental.sparse_split_multi(pg_mt)
+    pg_mt = pg_mt.filter_rows(hl.len(pg_mt.alleles) == 1)
```

Comment:
```suggestion
    pg_mt = pg_mt.filter_rows(hl.len(pg_mt.alleles) > 1)
```
sorry, gave you the wrong logic

Reply:
or 
```
pg_mt = pg_mt.filter_rows(hl.len(pg_mt.alleles) == 1, keep=False)
```